src/NLog.Targets.Seq/SeqTarget.cs

```diff
@@ -283,41 +295,37 @@ private async Task SendPayload(StringBuilder payload)
             // Even if no events are above `_minimumLevel`, we'll send a batch to make sure we observe minimum
             // level changes sent by the server.
 
-            using (var response = await _httpClient.SendAsync(request).ConfigureAwait(false))
+            using var response = await _httpClient.SendAsync(request).ConfigureAwait(false);
+            if ((int)response.StatusCode > 299)
             {
-                if ((int)response.StatusCode > 299)
-                {
-                    var data = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
-                    throw new WebException($"Received failed response {response.StatusCode} from Seq server: {data}");
-                }
+                var data = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                throw new WebException($"Received failed response {response.StatusCode} from Seq server: {data}");
+            }
 
-                if ((int)response.StatusCode == (int)HttpStatusCode.Created)
+            if ((int)response.StatusCode == (int)HttpStatusCode.Created)
```

Comment:
What about other status codes?

Reply:
I think this could be expanded if Seq starts to use different success status codes for ingest (e.g. `202/Accepted`), but currently it's correct/adequate.